src/structs.jl

```diff
@@ -176,6 +176,7 @@ mutable struct ScalarParameter <: MPSGEScalarVariable
         S = new(model,name,missing, value, description)
         add_variable!(model, S)
         fix(S, value)
+        set_start_value(S, value)
```

Comment:
Set the start value of a parameter. Solely because we want to call `start_value` before the first solve.